packages/suite-native-atoms/src/Modal/BottomModalContainer.tsx

```diff
@@ -18,8 +18,14 @@ const modalWithOverlayStyle = prepareNativeStyle(utils => ({
 
 export const BottomModalContainer = ({ children, isVisible, onClose }: ModalProps) => {
     const { applyStyle } = useNativeStyles();
+
     return (
-        <RNModal transparent visible={isVisible} onRequestClose={onClose}>
+        <RNModal
+            transparent
+            visible={isVisible}
+            onRequestClose={onClose}
+            style={{ paddingBottom: 100 }}
```

Comment:
are you sure this should be here?

Reply:
Fuuuck! Good catch. I was just testing different stuff and forgot to remove it.

Reply:
[5ca9f07](https://github.com/trezor/trezor-suite/pull/5579/commits/5ca9f079c5a74efd47d083b6b62fc98ebe7fe64e)